consensus/poet/core/sawtooth_poet/poet_consensus/consensus_state.py

```diff
@@ -63,6 +63,10 @@ class ConsensusState(object):
             claimed by all validators
     """
 
+    # MINIMUM_WAIT_TIME should be same as the value
+    # at ecall_CreateWaitTimer
```

Comment:
Maybe a better comment is that it should match the constant in the enclaves. That should include both the simulated enclave and the sgx enclave.